### PR TITLE
EWL-4479: SG2 | Create Press release page

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Jira Ticket**

- [EWL-4479: SG2 | Create Press release page](https://issues.ama-assn.org/browse/EWL-4479)

## Description

Adds a press page template. For now there are placeholders for the header and footer with a empty sidebar in this template. It will be used in the near future as a skeleton for components. Developers are expected to add their new components to this template when they are done.

## To Test

- [ ] gulp serve 
- [ ] http://localhost:3000/?p=pages-press
- [ ] Visually you will only see the header and footer for now. The right sidebar exists but it doesn't have anything it. 

## Visual Regressions
Update backstop.json with press release page test. Backstop was run initially to create reference images. These ref images will be the source of truth for all future alterations to this page.

## Relevant Screenshots/GIFs

screen shot 2018-02-05 at 11 54 11 am

## Remaining Tasks
n/a

## Additional Notes
n/a

### CODE REVIEW

File: styleguide/source/_patterns/05-pages/press.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Description
```
Comment (labbydev) [inline styleguide/source/_patterns/05-pages/press.md]: Update the title here as "Press Release" I worry that just "Press" will end up being confusing in the future or for content generators.

**Review by labbydev — COMMENTED** [review]
It all looks good from my POV. When running the tests locally it looks like everything is passing. I'm re-running on Travis to see if it will pass a second time. 

I do think we should update the name of Press to Press Release in case it becomes confusing in the future.

**Comment by chelsiejohnston** [comment]
Hey @designerneil - I know you made this PR before the standards were formally presented yesterday. That said, I'm posting this here and asking you to correct for learning purposes.

Please adjust the following in this PR before it gets further review:

(Per https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/develop/.github/CONTRIBUTING.md#pull-request-guidelines):

- [ ] **title formatting**

> The title must be a simple imperative sentence; examples,
>
>Good: Add Symfony code style documentation.
>Bad: Adding Symfony code style documentation. [bad because it is declarative]

> **The title must include the JIRA ticket number and the title must match the summary of the JIRA ticket.** Do not work on work that has not already been ticketed; examples,

> Good: [EWL-4421] Create News Header Organism
> Bad: Tools clean up

- [ ] **"Description" section context**

> state the solution: the context must lead into what you did to resolve the issue
> identify any follow up changes that will be needed and how that is going to be addressed, i.e. what does this PR not address?

(you pointed this out in the testing instructions, which is fine, but it definitely also belongs in the description where it can appear up-front to reviewers.)

An ideal description here would be something like "Adding the template here because we need to have the Page template file merged. That way everyone can add includes for smaller patterns to it once this PR is merged." The idea is to give a quick summary/context for people who might not have been in on our team's conversations about how to approach adding Page template files first, etc.

- [ ] **"Testing instructions" section content**

 > There will be nothing in the template for now. It's just a placeholder for things to come.

 _I_ get what you mean here, but please be more accurate in your statements in the future (there isn't "nothing" here as the footer and header placeholder sections are in place, so if I didn't know better I might get confused by that - stuff like that). 

 Please also add `gulp serve` as a first step (I know it's obvious, but adding it without also adding `composer install` or similar commands makes it clear to the reviewer that `gulp serve` is all that's needed to fully see the changes in the PR)

- [ ] **"Visual regressions" section**

Can you provide a little more context than just "Backstop visual testing"? I'm not sure what just stating this here is supposed to mean. Does it mean you added a test as part of this PR? That we have backstop testing in place generally? As a reviewer it should be clear what you're adding, and as a team member I should be able to quickly confirm that your PR meets the testing standards by looking at this section of the PR (that's why we added it to the template).

- [ ] **"Remaining Tasks" section**

Per the instructions at the top of the template, please put "N/A" in this section (and all sections) if they are not relevant to the work in the PR.

**Comment by chelsiejohnston** [comment]
Thanks @designerneil for fixing this up, it's a big improvement and it looks like the PR is almost there. 

That said - please take a more thorough look at the review and address some corrections that are still missing after your first edit: 

- [ ] **Title formatting**

> the title must match the summary of the JIRA ticket

Please update the title to match the summary of the JIRA ticket, in this case, "EWL-4479: SG2 | Create Press release page." Your title here is similar, but uses slightly different wording that could cause confusion to someone lacking our team's current contextual knowledge of the project.

- [ ] **Testing instructions**

Looks like this got missed: 

> Please also add `gulp serve` as a first step (I know it's obvious, but adding it without also adding `composer install` or similar commands makes it clear to the reviewer that `gulp serve` is all that's needed to fully see the changes in the PR)

Also, sorry for not pointing this out in my initial review, but: 

> The test had to run against itself because this page doesn't exist yet.

Please add more details here; this sentence doesn't really make sense. How can the page not exist if you're creating it in this PR? Or are you saying it doesn't exist on the live version of SG2? What does the test running "against itself" mean - does that mean the expected result is to fail? If that's the case, should I expect it to fail when I pull the branch down and test locally as well as on Travis in Github? Or for just one of these cases? etc.

(_I_ as Chelsie know the answers to these questions due to my contextual knowledge of being part of recent conversations, so no need to explain to me — just pointing out that if I didn't know the answers already, these are the questions I would need to see answered in this section of a PR in order to test/review properly.)

Please fix up the above and let me know when it's ready for review. Thanks!

**Comment by designerneil** [comment]
@chelsiejohnston thanks for reviewing my PR. the test portion has been updated

**Comment by chelsiejohnston** [comment]
@designerneil Almost there.

Please review my earlier message and make sure you have addressed all of the points.

Specifically: 

- Please update the title to match the summary of the JIRA ticket, in this case, "EWL-4479: SG2 | Create Press release page."
- Please also add gulp serve as a first step in the testing instructions

Thanks!

**Review by labbydev — APPROVED** [review]

**Pull request merged**